omnigibson/tasks/grasp_task.py

```diff
@@ -200,7 +200,7 @@ def reset(self, env):
     def _get_random_joint_position(self, robot):
         joint_positions = []
         joint_control_idx = th.cat([robot.trunk_control_idx, robot.arm_control_idx[robot.default_arm]])
-        joints = th.tensor([joint for joint in robot.joints.values()])
+        joints = th.tensor([joint for joint in robot.joints.values()], dtype=th.float32)
```

Comment:
This cant work, `joint` is an object not a float. This shouldn't even be a tensor. Here just get `joints = list(robot.joints.values())` and then get `arm_joints = [joints[i] for i in joint_control_idx.tolist()]`